modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/widget/WidgetManagerImpl.kt

```diff
@@ -33,27 +34,29 @@ class WidgetManagerImpl @Inject constructor(
 ) : WidgetManager {
 
     override fun updateWidget(podcast: Podcast?, playing: Boolean, playingEpisode: BaseEpisode?) {
-        if (Util.isAutomotive(context)) {
-            return
-        }
-
-        try {
-            val appWidgetManager = AppWidgetManager.getInstance(context)
-
-            val views = RemoteViews(context.packageName, R.layout.widget)
-            val widgetName = ComponentName(context, PodcastWidget::class.java)
-            if (playingEpisode == null) {
-                showPlayingControls(false, views)
-            } else {
-                showPlayingControls(true, views)
-                updateArtWork(podcast, playingEpisode, views, widgetName, context)
-                showPlayButton(playing, views)
-                updateSkipAmounts(views, settings)
+        when (Util.getAppPlatform(context)) {
```

Comment:
I'm suing a when here instead of just adding to the `if` statement we had before so that if/when we add more app platforms the compiler will force us to make a conscious decision here about whether we want to perform the widget setup instead of just defaulting to set up the widget like we were before.